pkg/server/container_create_test.go

```diff
@@ -805,6 +805,19 @@ func TestGenerateApparmorSpecOpts(t *testing.T) {
 			profile:    "",
 			privileged: true,
 		},
+		"should not return error if apparmor is unconfined when apparmor is not supported": {
+			profile:   unconfinedProfile,
+			disable:   true,
+			expectErr: true,
```

Comment:
The text is correct but the expectErr should be false here..

Reply:
Agree.